tools/rna_tools/compalignp/compalignp.xml

```diff
@@ -0,0 +1,50 @@
+<tool id="compalignp" name="Compalignp" version="1.0">
+    <description>Fractional identities between alignments</description>
+    <requirements>
+        <requirement type="package" version="1.0">compalignp</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Error occurred. Please check Tool Standard Error" />
+        <exit_code range=":-1" level="fatal" description="Error occurred. Please check Tool Standard Error" />
+    </stdio>
+    <command>
+<![CDATA[
+    compalignp
+        -t $rna_alignment_target
+        -r $rna_alignment_reference
+        $flag_verbose
+    > $output
+]]>
+    </command>
+    <inputs>
+        <param format="clustalw,stockholm" name="rna_alignment_target" type="data" label="Target RNA Alignment" help="(-t)" />
+        <param format="clustalw,stockholm" name="rna_alignment_reference" type="data" label="Reference RNA Alignment" help="(-r)" />
+        <param name="flag_verbose" type="boolean" truevalue="-v" falsevalue="" label="Verbose output" help="(-v)" />
```

Comment:
Is this just debugging output? If so this should not be exposed to the user.  If not the description could be enhanced, what kind of verbose information will be added and if this is useful?